std/testing/diff.ts

```diff
@@ -6,11 +6,12 @@ interface FarthestPoint {
   id: number;
 }
 
-export enum DiffType {
-  removed = "removed",
-  common = "common",
-  added = "added",
-}
+export const DiffType = {
```

Comment:
This change is necessary, because `string` is not assignable to enum types. Refer https://github.com/microsoft/TypeScript/issues/17690